util/qubes/qubes.lisp

```diff
@@ -0,0 +1,77 @@
+;;;; qubes.lisp
+
+(in-package #:qubes)
+
+;; Qubes module for StumpWM.
+;;
+;; Copyright (C) 2016 Johanna Abrahamsson
+;;
+;; Maintainer: Johanna Abrahamsson
+;;
+
+;;; Code:
+
+(defun alloc-color-on-current (color)
+  (stumpwm::alloc-color (current-screen) color))
+
+
+(setf *label-colors* (list (alloc-color-on-current "pink1")
```

Comment:
Because this introduces a new variable, it shouldn't use `setf`. In this case it should be `defparameter`.